robottelo/ui/base.py

```diff
@@ -596,6 +595,16 @@ def perform_action_chain_move(self, locator):
             )
         ActionChains(self.browser).move_to_element(element).perform()
```

Comment:
It is not needed, just trying to get more information. Thanks for the clarification.